src/EventStore.ClientAPI/CompatibilityMode.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using EventStore.ClientAPI.Common.Utils;
+
+namespace EventStore.ClientAPI {
+	/// <summary>
+	/// Compatibility mode factory.
+	/// </summary>
+	public static class CompatibilityMode {
+		/// <summary>
+		/// Creates a compatibility mode handler. If the string parameter is "disabled", the client will not run any
+		/// compatibility mode.
```

Comment:
I think this comment is confusing, if the string parameter is `disabled` the client will run v5 compatibility mode (to avoid breaking changes).

Reply:
Agree